src/components/contact-sheet.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import { useState } from "react";
+import {
+  Sheet,
+  SheetContent,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from "./ui/sheet";
+import { faUsers, faUserXmark } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { api } from "~/api/client";
+import { Button } from "./ui/button";
+import Image from "next/image";
+import ContactDialog from "./contact-dialog";
+
+export default function ContactSheet() {
+  const [open, setOpen] = useState(false);
+  const utils = api.useUtils();
+
+  const { data: contactList } = api.user.getContactList.useQuery();
+
+  const { mutateAsync: removeContact } = api.user.deleteContact.useMutation({
+    onSuccess() {
+      utils.user.getContactList.invalidate();
+    },
+  });
+
+  return (
+    <Sheet>
+      <SheetTrigger>
+        <FontAwesomeIcon icon={faUsers} onClick={() => setOpen(!open)} />
+      </SheetTrigger>
+      <SheetContent>
+        <SheetHeader>
+          <div className="flex justify-between px-4 pr-8">
+            <SheetTitle className="text-2xl">Contact List</SheetTitle>
+            <ContactDialog />
+          </div>
+          <div>
+            {contactList?.map((contact) => {
+              return (
+                <div key={contact.id} className="flex items-center gap-x-2">
+                  {contact.image ? (
```

Comment:
This is a lot of redundancy for basically only the src being changed. The `alt` attribute can only be "User profile picture" in both cases btw.